java/Makefile

```diff
@@ -830,7 +830,7 @@ JAVAC_JAVARI_ARGS = -processor org.checkerframework.checker.javari.JavariChecker
 JAVAC_IGJ_ARGS = -processor org.checkerframework.checker.igj.IGJChecker -implicit:class -Xlint:-processing -AskipDefs='${ALLSYSTEMS_SKIPDEFS}'
 ## Code in dcomp directory is not called directly; annotations would be useless.
 INDEX_SKIPDEFS = ^daikon\.dcomp\|^jtb\.
-INTERNING_SKIPDEFS = ^daikon\.dcomp\|^jtb\.
+INTERNING_SKIPDEFS = ^daikon\.Quant|^daikon\.dcomp|^jtb\.
```

Comment:
I don't understand Quant, so I didn't remove the warning suppressions in that file.